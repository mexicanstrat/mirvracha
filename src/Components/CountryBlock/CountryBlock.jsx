import React from 'react'
import './CountryBlock.css'

/**
 * Компонент отрисовки наименования страны

*/

export const CountryBlock = (props) => {
    return (
        <ul className="CountryBlock">
            <li>
                {props.country}
            </li>       
        </ul>
    )
}