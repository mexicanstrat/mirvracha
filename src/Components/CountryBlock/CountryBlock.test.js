import React from 'react'
import {shallow, mount, render} from 'enzyme'
import renderer from 'react-test-renderer'
import {CountryBlock} from './CountryBlock'

describe("CountryBlock", () => {
    it("render text", ()=> {
        const props = {
            country: "USSR",
        }
        const element = renderer.create(<CountryBlock {...props} />).toJSON();

        expect(element).toMatchSnapshot();

    })
})

