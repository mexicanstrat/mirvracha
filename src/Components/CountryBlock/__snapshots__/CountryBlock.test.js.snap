// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryBlock render text 1`] = `
<ul
  className="CountryBlock"
>
  <li>
    USSR
  </li>
</ul>
`;
